expcore/Mini_games.lua

```diff
@@ -207,8 +263,39 @@ function Mini_games.error_in_game(error_game)
     Mini_games.stop_game()
     game.print("an error has occured things may be broken, error: "..error_game)
 end
+
+local kick_all =
+function()
+    for i,player in ipairs(game.connected_players) do
+        game.kick_player(player,"You cant stay here")
+    end
+end
+
+Commands.new_command('kill_all','Command to stop a mini_game.')
```

Comment:
Confussing name since the command does not kill anyone, the description also doesnt make sense since this command doesnt stop the minigame. Suggestion: kick_all, Kicks all players from the game

Reply:
lmao yeah the discription is just a copy paste and ill rename it